connectors/src/connectors/zendesk/index.ts

```diff
@@ -208,9 +208,7 @@ export class ZendeskConnectorManager extends BaseConnectorManager<null> {
       return new Err(new Error("Connector not found"));
     }
 
-    const brandIds = await ZendeskBrandResource.fetchAllBrandIds({
-      connectorId,
-    });
+    const brandIds = await ZendeskBrandResource.fetchAllBrandIds(connectorId);
```

Comment:
Why not passing connector directly? To remove ambiguity.

Reply:
in the methods of the resources here the connectorId sometimes come from a `ConnectorResource`, sometimes from the `ConnectorManager` and are sometimes passed from a Temporal workflow to an activity, I opted for passing only IDs everywhere for consistency